index.d.ts

```diff
@@ -193,6 +193,33 @@ export interface Options {
 	```
 	*/
 	readonly width?: number;
+
+	/**
+	Set a fixed height for the box.
+
+	**Note*: This option will crop overflowing content.
+
+	@example
+	```
+	import boxen from 'boxen';
+
+	console.log(boxen('foo bar', {height: 5}));
+	// ┌───────┐
+	// │foo bar│
+	// │       │
+	// │       │
+	// └───────┘
+	```
+	*/
+	readonly height?: number;
+
+	/**
+	If possible, the box will fit all available space within the terminal.
+
+	**Note*: This option overrides the `width` and `height` options.
+
```

Comment:
```suggestion
```

Reply:
I think `width` and `height` should have higher priority. Then users can create a full-width or full-height box without difficulty.